src/pages/[movieId].js

```diff
@@ -0,0 +1,153 @@
+import styles from "@/styles/movie.module.scss";
+import { getMovie, getShowtimesForMovie } from "@/utils/dbFunctions";
+import dbConnect from "@/utils/dbConnect";
+
+export async function getServerSideProps(context) {
+  await dbConnect();
+
+  const { movieId } = context.query;
+  const movie = await getMovie(movieId);
+
```

Comment:
Looks good, but shouldn't there be a check here to see if the result of getMovie(movieId) is null or not and if null render a 404 page ?

Reply:
Excellent suggestion, there definitely should be 👍